docker/bin/msfvenom

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+# determine script path
+pushd $(dirname $(dirname $(dirname $(realpath $0)))) > /dev/null
```

Comment:
```realpath``` is not a available on all system POSIX systems e.g. macOS, sorry I missed this in your comment code stub.

```
docker/bin/msfconsole: line 4: realpath: command not found
usage: dirname path
usage: dirname path
usage: dirname path
docker/bin/msfconsole: line 4: pushd: no other directory
docker/bin/msfconsole: line 6: popd: directory stack empty
```